test/box-luatest/rtree_pagination_test.lua

```diff
@@ -0,0 +1,479 @@
+local server = require('luatest.server')
```

Comment:
Please add tests where:
1. Tuple which position is used was deleted from space.
2. New tuples are inserted before reading each next page.